creator-node/src/routes/tracks.js

```diff
@@ -504,7 +511,7 @@ module.exports = function (app) {
     req.params.streamable = true
 
     return getCID(req, res)
-  })
+  }))
```

Comment:
maybe I have a bad eye.... but where's paired opening for this parenthesis :O

Reply:
`handleResponse` ;)